src/app/user/organization-register/organization-register.page.html

```diff
@@ -1,36 +1,142 @@
 <form>
   <div class="auth-wrapper">
 
+    <!-- email -->
     <div class="auth-input">
-      <ion-input type="email" fill="solid" label="Имейл*" labelPlacement="floating" clearInput></ion-input>
+      <ion-input type="email" fill="solid" label="Имейл*" labelPlacement="floating" clearInput name="email" ngModel
+        #email="ngModel" clearInput required pattern="^[a-zA-Z0-9\._+\-]+@[a-zA-Z0-9\.]+\.[a-zA-Z]{2,}$"></ion-input>
     </div>
 
+    <ng-container *ngIf="email.touched">
+      <p class="auth-validation-error-message" *ngIf="email.errors?.['required']">
+        Полето е задължително за попълване!
+      </p>
+      <p class="auth-validation-error-message" *ngIf="email.errors?.['pattern']">
+        Моля въведете валиден имейл!
+      </p>
+
+      <!-- TODO: When server returns email already taken -->
+      <!-- <p class="auth-validation-error-message" *ngIf="emailError">
+        Този имейл вече е използван!
+      </p> -->
+    </ng-container>
+
+    <!-- password -->
     <div class="auth-input">
-      <ion-input type="password" fill="solid" label="Парола*" labelPlacement="floating" clearInput></ion-input>
+      <ion-input type="password" fill="solid" label="Парола*" labelPlacement="floating" clearInput name="password"
+        ngModel #password="ngModel" required minlength="8" clearInput
+        pattern="^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?!\s).+$"></ion-input>
```

Comment:
I think only min length and of course like 24 chars for max length is enough, without 1 small, 1 big letter, etc...